packages/react-next/package.json

```diff
@@ -30,7 +30,7 @@
     "start": "just-scripts dev:storybook",
     "start-test": "just-scripts jest-watch",
     "update-snapshots": "just-scripts jest -u",
-    "update-api": "just-scripts update-api"
+    "update-api": "just-scripts update-api && just-scripts update-api --configPath ./config/api-extractor-compat.json"
```

Comment:
we should add `api-extractor:verify ./config/api-extractor-compat.json` to `build` too?